data/beherabikram.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Bikram Behera",
+  "bio": "I am a MERNstack Developer. Programming enthusiast, touch typing & Love to learn new things",
+  "tags": ["Open Source", "MOngodb","Node js", "React js","Express js", "Java"],
+  "milestones": [
+    {
+      "title": "Started Contributing to Open Source",
+      "date": "Oct 2021",
+      "icon": "FaGithub",
+      "color": "black",
```

Comment:
Color key is no longer used, as the milestones are defaulted to Grey in our recent update! 

```suggestion
```